dotCMS/src/main/java/com/dotcms/security/apps/AppSecrets.java

```diff
@@ -30,6 +30,13 @@ public String getKey() {
         return secrets;
     }
 
+    public void destroy() {
```

Comment:
Maybe a test for the destroy, to check that is actually being destroyed?

Reply:
done.